lte/protos/session_manager.proto

```diff
@@ -740,7 +789,7 @@ enum SMSessionFSMState {
 //PDU session Establishment from AMF to SMF
 //SET MESSSAGE
 message M5GSMSessionContext {
-  bytes pdu_session_id = 1;   //One byte
+  uint32 pdu_session_id = 1;   //One byte
   RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
```

Comment:
leave a single space between // and text that follows everywhere.

Reply:
Resolved, fixed.